### ISSUE TITLE
PEAR.Commenting.ClassComment.MissingLinkTag doesn't recognize inline links

### ISSUE THREAD

**Issue by hollabaq86** [squizlabs/PHP_CodeSniffer#3382]
First of all, thanks everyone for creating and maintaining this excellent library 💛 

**Describe the bug**
I'm working on adding PHPDoc-themed sniffs to a library I help maintain, and I noticed that the rule `PEAR.Commenting.ClassComment.MissingLinkTag` is appearing in error messages for classes where I have [inline links](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.inlinelink.pkg.html) declared in descriptions. 

This is a simplified example of what I'm seeing:

**Code sample**
```php
/**
 * An example class
 * 
 * For more detailed information on this class, see {@link https://example.com our developer docs}. <br />
 */
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs -s test.php ...`
3. See error message displayed
```
Missing @link tag in class comment (PEAR.Commenting.ClassComment.MissingLinkTag)
```

**Expected behavior**
I don't expect to receive this particular error message since I'm providing inline link tags, but I'm not sure if this is a 🐛 or that the PEAR rule itself doesn't consider inline links valid.

As a workaround, I _could_ just declare a [formal link tag](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.link.pkg.html), but I think inline links provide a more semantically meaningful experience to the reader, which is why I thought it might be worth creating an issue here for review.

**Versions (please complete the following information):**
 - OS: debian buster
 - PHP: 8.0.5
 - PHPCS: ^3.0
 - Standard: PEAR

Thanks for your time!

**Comment by gsherwood** [comment]
The PEAR coding standards are quite old, so they use `@link` differently than you do. An example is shown here: https://pear.php.net/manual/en/standards.sample.php

But essentially, `@link` was used to link back to the homepage of the PEAR package, which is why it was required.

I'd recommend making changes to your ruleset or custom sniffs to not make `@link` a required tag in the way the PEAR standard does.

Hope that helps a bit.

**Comment by hollabaq86** [comment]
Thanks for the context, that’s super helpful. I’ll just go ahead and add this to the list of rules to exclude.

Feel free to close this issue!
